servers/rendering_server.cpp

```diff
@@ -847,8 +847,12 @@ void RenderingServer::mesh_surface_make_offsets_from_format(uint32_t p_format, i
 			}
 		}
 
-		r_offsets[i] = (*size_accum);
-		(*size_accum) += elem_size;
+		if (size_accum != nullptr) {
+			r_offsets[i] = (*size_accum);
+			(*size_accum) += elem_size;
+		} else {
+			r_offsets[i] = 0;
+		}
```

Comment:
Please review @godotengine/rendering @TokageItLab.

I don't know if that's the right fix but reading the code it does look like we're potentially dereferencing an uninitialized pointer here since it's only initialized for `RS::ARRAY_VERTEX`, `RS::ARRAY_COLORS` and `RS::ARRAY_BONES` but this loop goes through all of `RS::ARRAY_MAX`.